internal/http_cache/http_cache.go

```diff
@@ -117,8 +117,15 @@ func uploadCache(w http.ResponseWriter, r *http.Request, cacheKey string) {
 
 		if n > 0 {
 			chunkMsg := api.CacheEntry_Chunk{Chunk: &api.DataChunk{Data: readBuffer[:n]}}
-			uploadCacheClient.Send(&api.CacheEntry{Value: &chunkMsg})
-			bytesUploaded += n
```

Comment:
Shouldn't we keep this line in case `Send()` returns a `nil` error?

Reply:
Good catch. I initially rearrange code a bit and put it in the very end but forgot to restore. Fixed in 0eed3d6b23bd3b6cb8328a5d3dff4b0596098c9e